ui/src/timeMachine/actions/queries.ts

```diff
@@ -103,6 +105,58 @@ export const getOrgIDFromBuckets = (
   return get(bucketMatch, 'orgID', null)
 }
 
+//We only need a minimum of one bucket, function, and tag,
+export const getQueryFromFlux = (text: string) => {
```

Comment:
NOTE: flux team is working on fleshing this ability out for us a bit in the near future